### ISSUE TITLE
Is it possible to make the gdb frontend's command prompt by "(rr)" instead of "(gdb)"?

### ISSUE THREAD

**Issue by fitzgen** [mozilla/rr#1574]
Would be nice to differentiate. I used "(rr)" in some slides to make it clear that these commands were for use with rr, but it seems like a generally nice to have thing when actually using gdb as a frontend for rr.

Perhaps a good first bug?

I found this: https://www.sourceware.org/gdb/onlinedocs/gdb/Prompt.html

Is there a chance to run custom gdb commands before starting?

**Comment by rocallahan** [comment]
Yes, we already run some gdb commands on startup (see `gdb_rr_macros`). Easy first bug but tests would need to be fixed up.

**Comment by fitzgen** [comment]
Alright, I'll take a crack at it. Thanks for the pointer!
